syntax = "proto3";

package KeepAliveNs;

service KeepAliveService {
  rpc KeepAlive(Empty) returns (Empty) {}
}

message Empty {}

